index.js

```diff
@@ -97,121 +168,195 @@ Query.scale = function(_domain) {
       return result(false,true);
     };
     return res;   
-  };
+
+  });
 };
 
 Query.$valueFn = function(d) { return d; };
 
-// Pick returns a filtered subset of the records
-Query.where = function(_filters, $valueFn) {
-  var self = this;
-  return function $property(ref) {
-  
-    // Provide pipe delimited filtering
-    ref = ref.split(reSplitter).sort();
-    if (ref.length > 1)
-      // Solve for the first one, and then the remainder
-      return [ref[0],function(q) {
-        return [{q:q},'q.where.'+ref.slice(1).join(reSplitter),Object];
-      }];
+function generateEvaluateConditionFn(self, ast, $global, _filters, $valueFn, pipeOperation) {
+  if (ast.piped) {
```

Comment:
this is the new `where` and `where_not` support helper.  `where_not` was actually slightly different from `where` in terms of `true` and `false`